src/features/lightspeed/vue/views/ansibleCreatorUtils.ts

```diff
@@ -0,0 +1,422 @@
+        Uri.parse(destinationUrl),
+        ".venv",
+      ).fsPath;
+      let adeCommand = `cd ${destinationUrl} && ade install --venv ${venvPathUrl} --editable . --no-ansi`;
+      switch (collectionPayload.verbosity) {
+        case "low":
+          adeCommand += " -v";
+          break;
+        case "medium":
+          adeCommand += " -vv";
+          break;
+        case "high":
+          adeCommand += " -vvv";
+          break;
+      }
+
+      commandOutput += `\n\n-----------------------------------------ansible-dev-environment logs -----------------------------------------\n`;
+
+      await webView.postMessage({
+        command: "execution-log",
+        arguments: {
+          commandOutput:
+            "Collection scaffolding and environment installation in progress, please wait a few moments....\n",
+          logFileUrl: logFilePathUrl,
+          collectionUrl: destinationUrl,
+          status: "in-progress",
+        },
+      } as PostMessageEvent);
+      const adeVersion = await getADEVersion();
+      const exceedADEImVersion = semver.gte(adeVersion, ADE_ISOLATION_MODE_MIN);
+
+      if (exceedADEImVersion) {
+        adeCommand += " --im=cfg";
+        const { command, env } = withInterpreter(
+          extSettings.settings,
+          adeCommand,
+          "",
+        );
+        const adeExecutionResult = await runCommand(command, env);
+        commandOutput += adeExecutionResult.output;
+      } else {
+        commandOutput += `Collection could not be installed in editable mode.\n`;
+        commandOutput += `The required version of ansible-dev-environment (ade) for editable mode (using --isolation-mode=cfg) is ${ADE_ISOLATION_MODE_MIN}.\n`;
+        commandOutput += `The installed ade version on this system is ${adeVersion}\n`;
+        commandOutput += `Please upgrade to the latest version of ade for this feature.`;
+      }
+
+      console.debug("[ade] command: ", adeCommand);
+    }
+
+    await webView.postMessage({
+      command: "execution-log",
+      arguments: {
+        commandOutput: commandOutput,
+        logFileUrl: logFilePathUrl,
+        collectionUrl: isCollection ? destinationUrl : undefined,
+        projectUrl: isCollection ? undefined : destinationUrl,
+        status: ansibleCreatorCommandPassed,
+      },
+    } as PostMessageEvent);
+  }
+
+  public async isADEPresent(webView: vscode.Webview) {
+    const ADEVersion = await getBinDetail("ade", "--version");
+    if (ADEVersion === "failed") {
+      webView.postMessage({
+        command: "ADEPresence",
+        arguments: false,
+      } as PostMessageEvent);
+      console.debug(
+        "ADE not found in the environment. Disabling ADE features.",
+      );
+      return;
+    }
+    webView.postMessage({
+      command: "ADEPresence",
+      arguments: true,
+    } as PostMessageEvent);
+    console.debug("ADE found in the environment. Enabling ADE features.");
+    return;
+  }
+
+  public async getCollectionCreatorCommand(
```

Comment:
Name scheme should match the other getCreatorCommand methods: "getCollectionCreatorCommand" or "getCreatorCollectionCommand"